zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -41,12 +41,13 @@ <h3 class="new_h3" style="float:left">
       <div class="col-md-12">
         <h3 class="interpreter-title">{{setting.name}}
           <small>
-            <span ng-repeat="interpreter in setting.interpreterGroup"
+            <span style="display:inline-block" ng-repeat="interpreter in setting.interpreterGroup"
                   title="{{interpreter.class}}">
-              <span ng-show="$parent.$first && $first">%{{setting.name}} (default)</span>
-              <span ng-show="$parent.$first && !$first">, %{{interpreter.name}}</span>
-              <span ng-show="!$parent.$first && $first">%{{setting.name}}</span>
-              <span ng-show="!$parent.$first && !$first">, %{{setting.name}}.{{interpreter.name}}</span>
+              <span ng-show="!$first">, </span>
+              %<span ng-show="!$parent.$first || $first">{{setting.group}}</span
```

Comment:
In my understanding from doc, there are two cases that group name can be omitted.
1. when the interpreter group is selected as a default group.
2. first available interpreter in group.
For example when spark is not set to default group, user doesn't need to write `%spark.spark` but `%spark`.